### PR TITLE
Feature portfolio

### PR DESCRIPTION
**Pull request by Sammy3000**
## In this pull request I did the following:
- I added the navigation bar.
- I added the first content section which had the greetings and the paragraph lines.
- customized my readme file to include my social profilings.

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Sammy3000, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that the readme documentation is not properly customized. Kindly remove the descriptions given by the template.
image

Also, kindly ensure that the link to MIT file is not a broken one and that the note below the license is removed as it is only an instruction to the author of the repo. 
image

An appropriate title is also needed for the project. Kindly adjust this too.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.first-content {
+  height: 37.5rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(images/header-shapes\ mobile.png);
+}
```
Comment (Felix45) [inline style.css:38]: - [ Required ] Good job adding a background image to this `section` element, however on large screens it's not responsive. To make it responsive please consider giving it a `background-size: 100%;` to make the image responsive across small and large devices.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <p class="connect">let's connect</p>
+      <img
+        src="images/social media icons.png"
+        alt="social"
+        class="social-info"
+      />
```
Comment (Felix45) [inline index.html:34]: - [ Required ] Please consider adding a `ul` list with `li`'s that contain links `a` tag to the following social media pages twitter, linkedin, medium, github and angelist. In the coming months you'll use these social media pages for networking and job search so please add them.
### Example code
```HTML
<ul>
  <li><a href="http://twitter.com/yourhandle/"><img src="twitter_logo.png" /></a></li>
  <li><a href="http://linkedin.com/yourprofile/"><img src="linkedin_logo.png" /></a></li>
  <li><a href="http://facebook.com/yourprofile/"><img src="facebook_logo.png" /></a></li>
 <li><a href="http://medium.com/yourprofile/"><img src="medium_logo.png" /></a></li>
</ul>
```

File: README.md [hunk 3]
```diff
@@ -235,51 +94,15 @@ Example:
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
```
Comment (Felix45) [inline README.md:281]: [ required ]  On the readme file the link to the project LICENSE is very important in a project that has to do with intellectual property such as this one, please add a file named LICENSE to the project and make sure its contents are those of the [MIT License](https://choosealicense.com/licenses/mit/) on this [link](https://choosealicense.com/licenses/mit/). Remember to customize the `year` and `fullname` fields of the License and finally add the following lines to your `readme` file.

### Example markdown
```
## 📝 License <a name="license"></a>

This project is [MIT](./LICENSE) licensed.
````

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Sammy3000 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] All linter checks are setup and passing :white_check_mark: 
- [ ] Good descriptive pull request summary :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] Github flow is correctly used :white_check_mark: 

almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <nav class="main-nav">
+      <p class="logo">My Logo</p>
+      <img src="images/Union.png" alt="union" class="nav-img" />
+    </nav>
```
[the remark this answers is not in the document]
Comment (Sammy3000) [reply index.html]: @Felix45 noted.

File: index.html (hunk 2 above)
Comment (Sammy3000) [reply index.html:34]: @Felix45 I thought the social media icons should not link anywhere for now.

File: README.md (hunk 3 above)
Comment (Sammy3000) [reply README.md:281]: Thank you.

File: index.html (hunk 2 above)
Comment (Felix45) [reply index.html]: @Sammy3000 you should at the least have a Twitter, GitHub, Facebook and LinkedIn account

Comment (Sammy3000) [reply index.html]: @Felix45  okay noted thanks.

**Pull request merged**